dev/plugins/hu.elte.txtuml.stdlib/src/hu/elte/txtuml/stdlib/stdlib.uml

```diff
@@ -547,123 +547,124 @@
+      <ownedParameter xmi:id="_nJLtk2g8EeaTWu89rekDRA" name="return" direction="return">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Boolean"/>
       </ownedParameter>
     </ownedOperation>
-    <ownedOperation xmi:type="uml:Operation" xmi:id="_nJLtlGg8EeaTWu89rekDRA" name="eq" visibility="public" isStatic="true" method="_nJLta2g8EeaTWu89rekDRA">
-      <ownedParameter xmi:type="uml:Parameter" xmi:id="_nJLtlWg8EeaTWu89rekDRA" name="left">
+    <ownedOperation xmi:id="_nJLtlGg8EeaTWu89rekDRA" name="eq" visibility="public" isStatic="true" method="_nJLta2g8EeaTWu89rekDRA">
+      <ownedParameter xmi:id="_nJLtlWg8EeaTWu89rekDRA" name="left">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Real"/>
       </ownedParameter>
-      <ownedParameter xmi:type="uml:Parameter" xmi:id="_nJLtlmg8EeaTWu89rekDRA" name="right">
+      <ownedParameter xmi:id="_nJLtlmg8EeaTWu89rekDRA" name="right">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Real"/>
       </ownedParameter>
-      <ownedParameter xmi:type="uml:Parameter" xmi:id="_nJLtl2g8EeaTWu89rekDRA" name="return" direction="return">
+      <ownedParameter xmi:id="_nJLtl2g8EeaTWu89rekDRA" name="return" direction="return">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Boolean"/>
       </ownedParameter>
     </ownedOperation>
-    <ownedOperation xmi:type="uml:Operation" xmi:id="_nJLtmGg8EeaTWu89rekDRA" name="neq" visibility="public" isStatic="true" method="_nJLtbGg8EeaTWu89rekDRA">
-      <ownedParameter xmi:type="uml:Parameter" xmi:id="_nJLtmWg8EeaTWu89rekDRA" name="left">
+    <ownedOperation xmi:id="_nJLtmGg8EeaTWu89rekDRA" name="neq" visibility="public" isStatic="true" method="_nJLtbGg8EeaTWu89rekDRA">
+      <ownedParameter xmi:id="_nJLtmWg8EeaTWu89rekDRA" name="left">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Real"/>
       </ownedParameter>
-      <ownedParameter xmi:type="uml:Parameter" xmi:id="_nJLtmmg8EeaTWu89rekDRA" name="right">
+      <ownedParameter xmi:id="_nJLtmmg8EeaTWu89rekDRA" name="right">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Real"/>
       </ownedParameter>
-      <ownedParameter xmi:type="uml:Parameter" xmi:id="_nJLtm2g8EeaTWu89rekDRA" name="return" direction="return">
+      <ownedParameter xmi:id="_nJLtm2g8EeaTWu89rekDRA" name="return" direction="return">
         <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Boolean"/>
       </ownedParameter>
     </ownedOperation>
   </packagedElement>
+  <packagedElement xmi:type="uml:Class" xmi:id="_C8ZIwJYAEeeqkdBjqBM2Qg" name="hu.elte.txtuml.api.model.ModelClass"/>
```

Comment:
It seems this was the only intended change. Was the mass removal of the `xmi:type` attributes intentional, or was it caused by a graphical editor? If the latter, is this version semantically equivalent with the previous one (apart from the new `packagedElement`)?

Reply:
It cased by the graphical editor so I think this changes should be revetred.